content/reference-rippled.md

```diff
@@ -10955,6 +10957,182 @@ The response follows the [standard format](#response-formatting), with a success
 
 * Any of the [universal error types](#universal-errors).
 
+
+
+
+# Peer Protocol #
+
+Servers in the Ripple Consensus Ledger communicate to each other using the Ripple peer protocol, also known as RTXP. Peer servers connect via HTTPS and then perform an [HTTP upgrade](https://tools.ietf.org/html/rfc7230#section-6.7) to RTXP. (For more information, see the [Overlay Network article in the `rippled` repository](https://github.com/ripple/rippled/blob/906ef761bab95f80b0a7e0cab3b4c594b226cf57/src/ripple/overlay/README.md#handshake).)
```

Comment:
Quibble: break this link into two parts: one for the title of the article (i.e., [Overlay Network](https://github.com/ripple/rippled/blob/906ef761bab95f80b0a7e0cab3b4c594b226cf57/src/ripple/overlay/README.md#handshake)) and one for the `rippled` repo (i.e., [`rippled` repository](https://github.com/ripple/rippled)). The second part might be redundant, but I feel like it's a "nice" thing to do for the user.